packages/ui/src/SelectField.tsx

```diff
@@ -40,18 +46,35 @@ export type SelectFieldCoreStaticProps = {
   name: string
   onBlur?: (e: FocusEvent<HTMLElement>) => void
   error?: string
+  isMulti?: boolean
```

Comment:
Since `isMulti` is present in `SelectFieldCoreDynamicProps` we can remove it here.

Reply:
nice catch, thx!